src/freenet/node/Announcer.java

```diff
@@ -359,6 +359,15 @@ boolean enoughPeers() {
 			}
 			return true;
 		}
+		if (shutdownAnnouncement()) return true;
+
+		synchronized(timeGotEnoughPeersLock) {
+			timeGotEnoughPeers = -1;
+		}
+		return false;
+	}
+
+	private boolean shutdownAnnouncement() {
```

Comment:
“shutdownAnnouncement” sounds like a command but the method doesn’t do anything, and it returns a value. What is actually meant here? Is the shutdown announcement being shown? Should it be? Should announcements be shut down? Please rename the method in a way that makes it clearer what it does.